plugin/trino-sqlserver/src/test/java/io/trino/plugin/sqlserver/TestingSqlServer.java

```diff
@@ -96,7 +96,10 @@ public TestingSqlServer(String version, BiConsumer<SqlExecutor, String> database
         container.withUrlParam("database", databaseName)
                 // Instead of having a statement block forever (the default)
                 // Throws SQLServerException with SQLServerError code 1222  "Lock request time out period exceeded"
-                .withUrlParam("lockTimeout", Integer.toString(60 * 1000));
+                .withUrlParam("lockTimeout", Integer.toString(60 * 1000))
+                // TLS and certificate validation are on by default, and need
+                // to be disabled for tests.
+                .withUrlParam("encrypt", "false");
```

Comment:
We should set this param in `createContainer` to use it when verifying the connection in testcontainers.

Reply:
Move this down now ..